src/components/CalendarMonth/examples/4.select-dates.jsx

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash';
+import React from 'react';
+import moment from 'moment';
+import { DateUtils } from 'react-day-picker';
+import {
+	CalendarMonth,
+	Button,
+} from '../../../index';
+
+export default React.createClass({
+	getInitialState() {
+		return {
+			offset: 0,
+			selectedDays: [],
+			cursor: null,
+		};
+	},
+
+	handlePrev() {
+		this.setState({
+			offset: this.state.offset - 1,
+		});
+	},
+
+	handleNext() {
+		this.setState({
+			offset: this.state.offset + 1,
+		});
+	},
+
+	handleDayClick(date, { disabled }) {
+		if (disabled) {
+			return;
+		}
+
+		const {
+			selectedDays,
+		} = this.state;
+
+		this.setState({
+			selectedDays: _.xorWith(selectedDays, [date], DateUtils.isSameDay),
+			cursor: date,
+		});
+	},
+
+	handleDayMouseEnter(day, { disabled }) {
+		if (disabled) {
+			this.setState({
+				cursor: null,
+			});
+		} else {
+			this.setState({
+				cursor: day,
+			});
+		}
+	},
+
+	handleDayMouseLeave() {
+		this.setState({
+			cursor: null,
+		});
+	},
+
+	render() {
+		const {
+			selectedDays,
+			cursor,
+			offset,
+		} = this.state;
+
+		return (
+			<section>
+
+				<div style={{ display: 'flex' }}>
+					<Button onClick={this.handlePrev}>{'<'}</Button>
+
+					<CalendarMonth
+						monthOffset={offset}
+						selectedDays={selectedDays}
+						cursor={cursor}
+						onDayClick={this.handleDayClick}
+						onDayMouseEnter={this.handleDayMouseEnter}
+						onDayMouseLeave={this.handleDayMouseLeave}
+						disabledDays={DateUtils.isPastDay}
```

Comment:
it _is_ passed thru to the underlying `DayPicker`. not sure if we should document all the props that can be passed thru, or if we should cherry-pick props that we think will be significant?